.travis.yml

```diff
@@ -9,12 +10,18 @@ sudo: required
 before_install:
     - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update; fi
     - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo apt-get update -qq; fi
+    - if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then choco install msys2; fi
 
 install:
     - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install llvm; fi
     - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo apt-get install -qq flex bison clang lld-6.0; fi
+    - if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then cmd /c\ .travis_win_deps.bat; fi
+    # Build hang workaround
+    - if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then taskkill -IM "gpg-agent.exe" -F; fi
```

Comment:
Please provide more inforrmation about this workaround (like the URL which describes it) in the comment. Ideally there'd be a bugtracker issue we could track.

Reply:
I added the url. Unfortunately there doesn't seem to be a bugtracker - well, [there is one](https://github.com/travis-ci/travis-ci), but it's not supposed to be used anymore

Reply:
I'm assuming it's not running into this issue on #168 because I'm having to install via chocolaty which has it's own install scripts separate to what AppVeyor sets up.

Reply:
The URL was removed again?